### ISSUE TITLE
Line around Graphics object not on all sides when drawRect is used

### ISSUE THREAD

**Issue by MRVDH** [pixijs/pixi.js#5447]
### Expected Behavior
Draws line around every side of the object.

### Current Behavior
Line only drawn on three sides.
![capture](https://user-images.githubusercontent.com/6686385/52852408-cb390a80-3118-11e9-8203-c735b1fb8a09.PNG)

### Possible Solution
?

### Steps to Reproduce
```javascript
var testGraphic = new PIXI.Graphics();
testGraphic.beginFill(red).lineStyle(4, green);
testGraphic.drawRect(150, 150, 600, 200);
tool.stage.addChild(testGraphic);
```

### Environment

- **`pixi.js` version**: v5.0.0-rc
- **Browser & Version**: Chrome 72.0.3626.81 (Official Build) (64-bit)
- **OS & Version**: Windows 10
- **Running Example**: -

**Comment by ivanpopelyshev** [comment]
Confirmed, its differs in v4 and v5. `endFill()` doesn't help.

https://www.pixiplayground.com/#/edit/K45xTRXqfExRiZ6VUmx61

I suspected that something like that will appear, we rewrote Graphics class and I didn't test it enough.

**Comment by ivanpopelyshev** [comment]
`closePath` doesnt help too: https://www.pixiplayground.com/#/edit/QhH9QAw9gStIxvM2nUgYw

**Comment by ivanpopelyshev** [comment]
Can we close it? The fix was included in 5.0.0-rc2
